docs/phdata.rst

```diff
@@ -106,26 +108,25 @@ complete.
   - "smFRET-nsALEX" (2 excitation colors, 2 detection colors)
 
   New names can be created for different kind of measurements and we
-  encourage users to submit new names requests.
+  encourage users to submit new name requests.
 
-The field *measurement_type* represents the name of the specific measurement
-that is saved into the file. It is an important field that allows the sofware
-library that reads and saves Photon-HDF5 files to perform strict consistency
+The *measurement_type* field describes the type of measurement
+ saved within the file. It is an important field allowing sofware
+packages reading and saveing Photon-HDF5 files to perform consistency
 checks.
-In fact, each *measurement_type*, has an associated set of mandatory fields
-that must be present to assure the all the information needed to
+Each *measurement_type* has an associated set of mandatory fields
+which must be present to ensure that all information needed to
 unambiguously interpret the data is present.
-For example, for a 2-color smFRET measurement the software can check if
-the specification of which detector represents the donor or acceptor channel
-is present. If not present, it can throw and error or warn the user so that
-this important information can be added before saving the file.
-This prevents the accidental creation of incomplete or inconsistent files.
+For example, for a 2-color smFRET measurement, a software package creating a file should check that
+the association between detector and donor or acceptor channel
+is present. If absent, the software package should warn the user in order that
+this information is added before saving the file (**that doesn't make much sense in a file format description. Either the software is designed to write an incomplete data set, and then it is not compliant, or it is designed properly and it will not write a file if there are missing fields**).
 
```

Comment:
Yes, you are right. It is just the rational of having such a field, it should be moved elsewhere though.